templates/en/docs/routing/_hosts.md

```diff
@@ -0,0 +1,33 @@
+## Hosts
+
+<%= sinceVersion("0.18.2") %>
+
+Buffalo apps also support grouping of end-points by host. Host creates a new group that matches the domain passed. This is useful for creating groups of end-points for different domains or subdomains.
```

Comment:
```suggestion
Buffalo apps also support grouping of end-points by host. `Host` creates a new group that matches the domain passed. This is useful for creating groups of end-points for different domains or subdomains.
```

Reply:
Updated in https://github.com/gobuffalo/docs/pull/605/commits/d4b35fe13f94431f3aa51537735d4e1fd5d59348